### PR TITLE
Add kube2pulumi demo page

### PR DESCRIPTION
**Pull request by EvanBoyle**
Add a `kube2pulumi` demo page. This branch is based off of https://github.com/pulumi/docs/pull/3776, but I've removed the actual `/tf2pulumi` page and route since we're not quite ready with a launch plan. Given that, the relevant commit to review is: https://github.com/pulumi/docs/commit/e2388117fef32e227c3f6104dd5e300303aed603

You will notice `kube2pulumi.md/html` are basically a copy of the equivalent tf2pulumi files. This is an intentional choice due to how close we are to launch. Happy to take a look at cleaning some of this up next week after all of the KubeCon dust settles. 

Closes https://github.com/pulumi/home/issues/1028

### CODE REVIEW

File: content/topics/kube2pulumi.md [hunk 1]
```diff
@@ -0,0 +1,639 @@
...
+
+    // Add some "waiting" touches.
+    $(document.body).css({ "cursor": "wait" });
+    $("#terraform-code, #terraform-url").css({ "cursor": "wait" });
+    $("pulumi-chooser[type='language'] > ul > li > a").css({ "cursor": "wait" });
+    addLanguageFile(languageTextbox, "…", "…");
+
+    // Post to the endpoint and then, afterwards, add the result to the textbox.
+    let post = {
+        url: "https://k3wz4qdpci.execute-api.us-west-2.amazonaws.com/stage/convertKube",
```
Comment (EvanBoyle) [inline content/topics/kube2pulumi.md:297]: I will update this URL before merging once we get https://github.com/pulumi/home/pull/1042 deployed.

**Comment by EvanBoyle** [comment]
the route is `/kube2pulumi` so the most recent preview would be:

http://pulumi-docs-origin-pr-3923-c09a6bdb.s3-website.us-west-2.amazonaws.com/kube2pulumi

File: content/topics/kube2pulumi.md [hunk 2]
```diff
@@ -0,0 +1,639 @@
...
+
+{{< chooser input-kind "code,url,upload" >}}
+
+{{% choosable input-kind "code" %}}
+
+<p class="m-0 -mt-4 p-2 bg-purple-300 text-white font-bold font-mono font-xs"
+    style="font-size: 0.75rem !important; color: #fff !important">kube.yaml</p>
+<textarea id="terraform-code" rows="27"
+    class="w-full px-6 py-4 text-gray-700 text-sm font-mono overflow-y-scroll overflow-x-hidden whitespace-pre"
+    title="Enter a single-file HCL program's text; see the 'UPLOAD' tab for multi-file programs">
```
Comment (lblackstone) [inline content/topics/kube2pulumi.md:40]:
```suggestion
    title="Enter a single-file YAML manifest's text; see the 'UPLOAD' tab for multi-file programs">
```

**Comment by lblackstone** [comment]
The URL and upload tabs weren't working when I tried, but the YAML input is 🔥

**Comment by EvanBoyle** [comment]
> The URL and upload tabs weren't working when I tried, but the YAML input is 🔥

We found that this was due to some unexpected comment formatting. url/upload has the same behavior as the text input

**Comment by lukehoban** [comment]
Just tried using URL and content for https://raw.githubusercontent.com/pulumi/pulumi-kubernetes-operator/master/stack-examples/yaml/s3_bucket_stack.yaml and both lead to the very cryptic:

```
[31mError[0m: Missing attribute separator

  on pcl-859051780.pp line 40:
  40: config = {
  41: aws:region [1;4m=[0m "us-east-2"

Expected a newline or comma to mark the beginning of the next attribute.
```

**Comment by lukehoban** [comment]
> Just tried using URL and content for https://raw.githubusercontent.com/pulumi/pulumi-kubernetes-operator/master/stack-examples/yaml/s3_bucket_stack.yaml and both lead to the very cryptic

Opened https://github.com/pulumi/kube2pulumi/issues/36 to track this since it seems just an issue in core kube2pulumi.

**Pull request merged**